### DISCUSSION TITLE
Using this project on other boards like Genesys ZU-3EG

### DISCUSSION THREAD

**Discussion by fetchingcat** [eugene-tarassov/vivado-risc-v#207] [General]
How difficult would it be to make this work on other boards?  I've been looking at the genesys 2 but it's 8+ years old now and I'm wondering about Genesys ZU-3EG.

**Comment by eugene-tarassov** [comment]
> How difficult would it be to make this work on other boards?

It depends on experience.

> I'm wondering about Genesys ZU-3EG.

Genesys ZU-3EG is not suitable for experiments with RISC-V. It has very small FPGA, only 71K LUTs versus 200K LUTs on Genesys 2.

**Reply by fetchingcat** [reply]
I see.  I thought it had 154K logic cells.  I've had your project run on my nexys a7 and it's great but want to dig into the BOOM project.  Ive designed some hobby projects and used various IP packages and self built linux but always lots to learn.  I'm not familiar yet with some of the tooling you have used.  Very cool project and thank you for sharing.

**Reply by eugene-tarassov** [reply]
> I see. I thought it had 154K logic cells.

Yes, it has 154K system logic cells, which is same as 71K LUTs - these are two different ways to measure FPGA capacity.

In case you decide to add support for another board, look into [vivado-risc-v/board](https://github.com/eugene-tarassov/vivado-risc-v/tree/master/board) directory, use existing subdirectories as examples of board support packages.

**Reply by fetchingcat** [reply]
Ok, thank you.  The terminology is a bit confusing.  I would have assumed at that price and ultrascale+ that it didn't have that small of an FPGA.  I guess all the die space is used by the arm cores which I'm less interested in.  Good to know!  I'll look through the board files as I'm learning how your project works.

**Reply by fetchingcat** [reply]
I ultimately found the data sheet and confirmed exactly what you said.  I ended up getting the Genesys 2 after all.  Thanks for the help and guidance and saving me from an FPGA that wouldn't do what I wanted.  Now, off to play with BOOM.
